python/pipeline/vreso.py

```diff
@@ -0,0 +1,175 @@
+import scanreader
+import numpy as np
+import datajoint as dj
+from pipeline.exceptions import PipelineException
+from pipeline import reso, experiment, mice, notify, shared
+from pipeline.utils import galvo_corrections, signal, quality, mask_classification, performance, h5
+
+# external storage place
+dj.config['external'] = {'protocol': 'file',
+                         'location': '/mnt/dj-stor01/pipeline-externals'}
+
+schema = dj.schema("pipeline_vreso", locals(), create_tables=False)
+CURRENT_VERSION = 1
+
+
+@schema
+class Amplifier(dj.Lookup):
+    definition = """ # Electrophysiology rig setup
+    amplifier                     : varchar(32)         # name and model of patch rig
+    """
+    contents = [
```

Comment:
we usually add a description field (not as primary key) with some text about what the entry means. Unless the name is more than enough to describe it (like "red")